src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/InfiniteScrollList.xaml.cs

```diff
@@ -290,6 +308,111 @@ private void LoadItems(PackageItemListViewModel selectedPackageItem, Cancellatio
             });
         }
 
+        internal void FilterItems(ItemFilter itemFilter, CancellationToken token)
```

Comment:
Basically `LoadItems` but  instead of hitting refresh logic, it does UI filtering. May end up removing some logic here like loadingStatus and logging.